components/camel-pubnub/src/main/java/org/apache/camel/component/pubnub/PubNubConsumer.java

```diff
@@ -131,6 +138,41 @@ public void presence(PubNub pubnub, PNPresenceEventResult presence) {
             }
         }
 
+        @Override
+        public void signal(@NotNull PubNub pubnub, @NotNull PNSignalResult pnSignalResult) {
+            System.out.println("A");
```

Comment:
Are you sure about this println?

Reply:
ops!, not really :) anyway tests that are present in pubnub do not use these features